### PR TITLE
[#14] Make MsgPythonShell more robust

### PR DESCRIPTION
**Pull request by petrieh**
Make MsgPythonShell to stand message losts and corruptions.

### CODE REVIEW

File: src/crl/interactivesessions/shells/msgpythonshell.py [hunk 1]
```diff
@@ -44,36 +33,59 @@ class ExecCommandError(InteractiveSessionError):
 
 
 class MsgPythonShell(RawPythonShell):
+    """
+    timeout=RawPythonShell.short_timeout)
+    """
```
Comment (vereszol) [inline src/crl/interactivesessions/shells/msgpythonshell.py]: This docstring should be deleted

File: src/crl/interactivesessions/shells/remotemodules/msgcaches.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            self._missing_nbrs |= set(range(self._max + 1, i))
+            self._max = i
+        elif self._min < i < self._max and i in self._missing_nbrs:
+            self._missing_nbrs.remove(i)
+        elif i < self._min:
+            self._missing_nbrs |= set(range(i + 1, self._min))
+            self._min = i
+
+    def __contains__(self, i):
+        return self._min <= i and i <= self._max and i not in self._missing_nbrs
```
Comment (petrieh) [inline src/crl/interactivesessions/shells/remotemodules/msgcaches.py]: Can be made shorter    `self._min <= i and i <= self._max`--> self._min <= i <= self._max

File: tests/shells/remotemodules/test_msgs.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+from crl.interactivesessions.shells.remotemodules.msgs import MsgBase
+
+
```
Comment (petrieh) [inline tests/shells/remotemodules/test_msgs.py]: Missing copyright

File: src/crl/interactivesessions/RunnerHandler.py [hunk 4]
```diff
@@ -10,15 +10,15 @@
...
 import threading
+import fcntl
 from contextlib import contextmanager
 from functools import wraps
 
 
 __copyright__ = 'Copyright (C) 2019, Nokia'
 
-LOGGER = logging.getLogger(__name__)
+TOKEN = '+7_<sf80UBsd%smz'
```
Comment (petrieh) [inline src/crl/interactivesessions/RunnerHandler.py]: should not contain duplicates

**Pull request merged**